user_guide_src/source/concepts/autoloader.rst

```diff
@@ -10,7 +10,7 @@ hard-coding that location into your files in a series of ``requires()`` is a mas
 headache and very error-prone. That's where autoloaders come in.
 
 CodeIgniter provides a very flexible autoloader that can be used with very little configuration.
-It can locate individual non-namespaced classes, namespaced classes that adhere to
+It can locate individual namespaced classes that adhere to
 `PSR-4 <https://www.php-fig.org/psr/psr-4/>`_ autoloading
 directory structures, and will even attempt to locate classes in common directories (like Controllers,
 Models, etc).
```

Comment:
`and will even attempt to locate classes in common directories (like Controllers, Models, etc)`
Does this make sense? I think this is already covered by PSR4 as we enforce controllers/models match their paths?

Reply:
No, I removed.